printer_mods/al3ph/led_mount/README.md

```diff
@@ -0,0 +1,5 @@
+Led holder for WS2812 type LED strings, includes channel for data line to loop back for second strip.
+
+Mounts on top extrusion, using 2 M3 bolts per strip, two strips required per side, (so they can be printed on V0).

```

Comment:
Can you add a picture of how it mounts?  Is it on top of the top extrusion?  Does it interfere with the top hat?

Reply:
Updated readme and added supports